### ISSUE TITLE
[Bug] Appflowy crashed when try to open it

### ISSUE THREAD

**Issue by buxingzhe** [AppFlowy-IO/AppFlowy#4808]
### Bug Description

Appflowy exits at startup.
Here are the stack traces:

`C:\Applications\AppFlowy>AppFlowy.exe --verbose

C:\Applications\AppFlowy>  2024-03-04 10:12:38  INFO flowy_core: 🔥AppFlowy Configuration { app_version: "0.5.0", storage_path: "C:\\Workspace\\Notes\\AppFlowyDataDoNotRename_aHR0cHM6Ly9iZXRhLmFwcGZsb3d5LmNsb3Vk", application_path: "C:\\Users\\nealh\\AppData\\Roaming\\io.appflowy\\AppFlowy\\data", base_url: "https://beta.appflowy.cloud", ws_url: "wss://beta.appflowy.cloud/ws", gotrue_url: "https://beta.appflowy.cloud/gotrue" }
    at flowy-core\src\lib.rs:82

  2024-03-04T02:12:38.121887Z  INFO flowy_core: 🔥AppFlowy Configuration { app_version: "0.5.0", storage_path: "C:\\Workspace\\Notes\\AppFlowyDataDoNotRename_aHR0cHM6Ly9iZXRhLmFwcGZsb3d5LmNsb3Vk", application_path: "C:\\Users\\nealh\\AppData\\Roaming\\io.appflowy\\AppFlowy\\data", base_url: "https://beta.appflowy.cloud", ws_url: "wss://beta.appflowy.cloud/ws", gotrue_url: "https://beta.appflowy.cloud/gotrue" }
    at flowy-core\src\lib.rs:82

  2024-03-04 10:12:38  INFO flowy_core: 💡System info: Some("Windows 11 Enterprise")
    at flowy-core\src\lib.rs:83

  2024-03-04T02:12:38.122463Z  INFO flowy_core: 💡System info: Some("Windows 11 Enterprise")
    at flowy-core\src\lib.rs:83

  2024-03-04 10:12:38  INFO flowy_user::user_manager::manager: init user session: 272288429763465216:ced9455a-e10d-4dd8-88eb-9bafb2d54ea5@appflowy.io
    at flowy-user\src\user_manager\manager.rs:148

  2024-03-04T02:12:38.325774Z  INFO flowy_user::user_manager::manager: init user session: 272288429763465216:ced9455a-e10d-4dd8-88eb-9bafb2d54ea5@appflowy.io
    at flowy-user\src\user_manager\manager.rs:148

  2024-03-04 10:12:38  INFO flowy_user::services::db: open collab db for user 272288429763465216 at path: "C:\\Workspace\\Notes\\AppFlowyDataDoNotRename_aHR0cHM6Ly9iZXRhLmFwcGZsb3d5LmNsb3Vk/272288429763465216\\collab_db"
    at flowy-user\src\services\db.rs:203

  2024-03-04T02:12:38.326210Z  INFO flowy_user::services::db: open collab db for user 272288429763465216 at path: "C:\\Workspace\\Notes\\AppFlowyDataDoNotRename_aHR0cHM6Ly9iZXRhLmFwcGZsb3d5LmNsb3Vk/272288429763465216\\collab_db"
    at flowy-user\src\services\db.rs:203

  2024-03-04 10:12:38  INFO flowy_user::services::db: open collab db to validate data integration for user 272288429763465216
    at flowy-user\src\services\db.rs:382

  2024-03-04T02:12:38.400362Z  INFO flowy_user::services::db: open collab db to validate data integration for user 272288429763465216
    at flowy-user\src\services\db.rs:382

  2024-03-04 10:12:38  INFO flowy_user::user_manager::manager: Prepare user data migration
    at flowy-user\src\user_manager\manager.rs:239

  2024-03-04T02:12:38.400741Z  INFO flowy_user::user_manager::manager: Prepare user data migration
    at flowy-user\src\user_manager\manager.rs:239

  2024-03-04 10:12:38  INFO flowy_user::services::db: Current backup: 10
    at flowy-user\src\services\db.rs:348
    in flowy_user::services::db::backup_collab_db

  2024-03-04T02:12:38.402404Z  INFO flowy_user::services::db: Current backup: 10
    at flowy-user\src\services\db.rs:348
    in flowy_user::services::db::backup_collab_db

thread '<unnamed>' panicked at library\std\src\time.rs:433:33:
overflow when subtracting duration from instant
stack backtrace:
   0:     0x7ffc0072ed8a - <unknown>
   1:     0x7ffbff87a83d - <unknown>
   2:     0x7ffc0071fa01 - <unknown>
   3:     0x7ffc0073065a - <unknown>
   4:     0x7ffc0072ff49 - <unknown>
   5:     0x7ffc00730d7b - <unknown>
   6:     0x7ffc00730885 - <unknown>
   7:     0x7ffc007307c9 - <unknown>
   8:     0x7ffc007307b4 - <unknown>
   9:     0x7ffc00d61c97 - <unknown>
  10:     0x7ffc00d622b2 - <unknown>
  11:     0x7ffbffa0ca16 - <unknown>
  12:     0x7ffbffa05c47 - <unknown>
  13:     0x7ffbff4c4b36 - <unknown>
  14:     0x7ffbff4c9a56 - <unknown>
  15:     0x7ffbff4cfe6b - <unknown>
  16:     0x7ffbff4cb289 - <unknown>
  17:     0x7ffbff4b67cb - <unknown>
  18:     0x7ffbff525666 - <unknown>
  19:     0x7ffbff51a391 - <unknown>
  20:      0x218cd6fa8e0 - <unknown>
  21:      0x218cd6fa87f - <unknown>
  22:      0x218cd6fa786 - <unknown>
  23:      0x218cd6fa6f4 - <unknown>
  24:      0x218cd6fa5f2 - <unknown>
  25:      0x218cd8c068a - <unknown>
  26:      0x218cca2d64c - <unknown>
  27:      0x218cca2db28 - <unknown>
  28:      0x218cca2da04 - <unknown>
  29:      0x218cca2c1c8 - <unknown>
  30:      0x218cca2be32 - <unknown>
  31:      0x218cd8c0abc - <unknown>
  32:      0x218cca2a1ad - <unknown>
  33:      0x218cd8c068a - <unknown>
  34:      0x218cca2d64c - <unknown>
  35:      0x218cca2db28 - <unknown>
  36:      0x218cca2da04 - <unknown>
  37:      0x218cca2c1c8 - <unknown>
  38:      0x218cca2be32 - <unknown>
  39:      0x218cd8c0abc - <unknown>
  40:      0x218cca2a1ad - <unknown>
  41:      0x218cd8c068a - <unknown>
  42:      0x218cca2d64c - <unknown>
  43:      0x218cca2db28 - <unknown>
  44:      0x218cca2da04 - <unknown>
  45:      0x218cca2c1c8 - <unknown>
  46:      0x218cca2be32 - <unknown>
  47:      0x218cca2bda0 - <unknown>
  48:      0x218cca1c7e7 - <unknown>
  49:      0x218cca1c6be - <unknown>
  50:      0x218cca1c687 - <unknown>
  51:      0x218cca14522 - <unknown>
  52:     0x7ffc01e95b85 - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  53:     0x7ffc01e96143 - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  54:     0x7ffc01e96565 - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  55:     0x7ffc0232b4ee - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  56:     0x7ffc01de4ef2 - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  57:     0x7ffc017c21f1 - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  58:     0x7ffc017cb60a - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  59:     0x7ffc017c20cf - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  60:     0x7ffc017c6518 - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  61:     0x7ffc024af632 - FlutterDesktopTextureRegistrarMarkExternalTextureFrameAvailable
  62:     0x7ffc7921257d - BaseThreadInitThunk
  63:     0x7ffc79d8aa58 - RtlUserThreadStart
`

### How to Reproduce

Just try to open Appflowy.

### Expected Behavior

Open successfully

### Operating System

Windows

### AppFlowy Version(s)

0.5.0

**Comment by LucasXu0** [comment]
Hi, @buxingzhe. Did you perform any other operations before launching the app?

**Comment by JettChen12** [comment]
same here

Platform:  
windows10

How to Reproduce: 
The first time I opened the it I selected offline mode and then crashed, every time I opened this app after that it stopped at a weird page then crashed again, even I reinstalled it.

Screenshots:
![图片](https://github.com/AppFlowy-IO/AppFlowy/assets/121156989/a7959bf3-c910-4097-be04-b87f0b175395)

**Comment by LucasXu0** [comment]
Hi there. Here's the pre-release version. Can you try installing it and check if the crash has been fixed?

[AppFlowy-0.5.1-windows-x86_64.exe.zip](https://github.com/AppFlowy-IO/AppFlowy/files/14505170/AppFlowy-0.5.1-windows-x86_64.exe.zip)

**Comment by JettChen12** [comment]
> Hi there. Here's the pre-release version. Can you try installing it and check if the crash has been fixed?
> 
> [AppFlowy-0.5.1-windows-x86_64.exe.zip](https://github.com/AppFlowy-IO/AppFlowy/files/14505170/AppFlowy-0.5.1-windows-x86_64.exe.zip)

Thanks for the quick reply. It's working fine on my machine.

**Comment by LucasXu0** [comment]
Good news! I appreciate that. We will release a new version that includes this bug fix asap.

**Comment by Leo5661** [comment]
I am still facing the same problem while trying to open it.

Environment:
- AppFlowy version (0.5.0)
- Windows 11

How to Reproduce:
I attempted to login using GitHub but the app got stuck on the loading screen, so I closed and reopened it, but it still froze and crashed. I tried reinstalling the app, but the issue persisted.

screenshot
![appFlowy_bug](https://github.com/AppFlowy-IO/AppFlowy/assets/35341767/1e4309ed-f1a0-4e48-ba26-ab3e99c58af8)
